### PR TITLE
Add Profile Music

### PR DESCRIPTION
**Pull request by Torpidwaters**
Do not push live without first adding the database column "music" of type TEXT in the existing table "users"

### CODE REVIEW

**Comment by panpawn** [comment]
There's no way this should be a thing, for several reasons.
1. Profile on Wisp is already getting pretty cluttered looking, and it already needs to redesigned as it is before we go adding more stuff to clutter it.
2. What about the staff having to listen to each and every "music" that is bought to add to their profile? That would be pretty annoying, since I imagine a lot of people would be buying this.
3. Dynamic sound hosting companies are a thing... even if someone listened to a song before they added it, the user could later change it to something inappropriate and/or malicious.

tldr; there's a lot of issues with this kind of feature, and this implementation, and there are other things that should be worked on for the display of profile _before_ we go adding more stuff to clutter it.

**Comment by Torpidwaters** [comment]
1 and 2 were already discussed before I started the coding and upper staff was fine with it, and originally were the ones who asked for it, so I got the go-ahead.  The third point is very valid though, so at the very least that would need discussion.  Thanks for the input panpawn, and good catch
